specs/mediawiki/v1/content.yaml

```diff
@@ -94,6 +95,24 @@ paths:
             $ref: '#/definitions/problem'
       x-backend-request:
         uri: /{domain}/sys/page_revisions/page/{title}
+      x-amples:
```

Comment:
It would be good to put another example that fetches the latest rev directly from storage after this one.